airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/client.py

```diff
@@ -2,34 +2,29 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-import csv
 import io
 import logging
-import os
-import re
-import socket
-import stat
-from datetime import datetime
-from typing import Any, Dict, List, Mapping, Tuple
 
 import backoff
-import numpy as np
-import pandas as pd
 import paramiko
+from airbyte_cdk.utils import AirbyteTracedException
+from airbyte_protocol.models import FailureType
 from paramiko.ssh_exception import AuthenticationException
 
 # set default timeout to 300 seconds
 REQUEST_TIMEOUT = 300
 
 logger = logging.getLogger("airbyte")
 
-File = Dict[str, Any]
+
+def handle_backoff(details):
+    logger.warning("SSH Connection closed unexpectedly. Waiting {wait} seconds and retrying...".format(**details))
 
 
 class SFTPClient:
-    _connection = None
+    _connection: paramiko.SFTPClient = None
 
-    def __init__(self, host, username, password=None, private_key=None, port=None, timeout=REQUEST_TIMEOUT):
+    def __init__(self, host: str, username: str, password: str = None, private_key=None, port: int = None, timeout=REQUEST_TIMEOUT):
```

Comment:
Please use `Optional[str]` if default is present and add annotation for the `timeout` param